static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+      "title": "PhotonIQ Performance Proxy API",
+      "description": "API documentation for the PhotonIQ P3 API Service",
+      "version": "2.2.5"
+    },
+    "servers": [
+      {
+        "url": "https://demo-p3.photoniq.macrometa.io"
+      }
+    ],
+    "tags": [
+      {
+        "name": "Version"
+      },
+      {
+        "name": "Health"
+      },
+      {
+        "name": "Policies"
+      },
+      {
+        "name": "Validations"
+      },
+      {
+        "name": "Cache"
+      },
+      {
+        "name": "PPM"
+      },
+      {
+        "name": "Authentication"
+      },
+      {
+        "name": "Metrics"
+      },
+      {
+        "name": "Telemetry",
+        "description": "(example can change)"
+      },
+      {
+        "name": "Usage",
+        "description": "(example can change)"
+      }
+    ],
+    "components": {
+      "securitySchemes": {
+        "ApiKeyAuth": {
+          "type": "apiKey",
+          "in": "header",
+          "name": "Authorization"
+        }
+      },
+      "schemas": {
+        "CreatePolicyRequest": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "Name of the policy to be created.",
+              "example": "Test_Policy"
+            },
+            "pageType": {
+              "type": "string",
+              "description": "Type of page to be optimized.",
+              "example": "Home"
+            },
+            "urlPattern": {
+              "type": "string",
+              "description": "Url pattern to group same pages under similar set of optimizations.",
+              "example": "^/us/.*$"
+            },
+            "serverUrl": {
+              "type": "string",
+              "description": "Origin/server URL from where the html content needs to be fetched to run optimizations.",
+              "example": "something.com"
+            },
+            "deviceType": {
+              "type": "string",
+              "description": "Type of device the optimizations are intended to be applied.",
+              "example": "Desktop"
+            },
+            "optimizationLevel": {
+              "type": "string",
+              "description": "Optimization category to decide which optimizations needs to be applied.",
```

Comment:
I am not sure how this description describes what the field is for.  I don't know how I can use optimizationLevel?
Is there a list of optimizations?  What do I put here?